modules/andes-core/broker/src/main/java/org/wso2/andes/configuration/enums/AndesConfiguration.java

```diff
@@ -432,21 +432,21 @@
      * (- slots which have a aged more than 'messageAccumulationTimeout')
      */
     PERFORMANCE_TUNING_SUBMIT_SLOT_TIMER_PERIOD (
-            "performanceTuning/slots/timerPeriod", "3000", Integer.class, PERFORMANCE_TUNING_SUBMIT_SLOT_TIMEOUT),
+            "performanceTuning/slots/maxSubmitDelay", "3000", Integer.class, PERFORMANCE_TUNING_SUBMIT_SLOT_TIMEOUT),
```

Comment:
Don't we need to change the name of the variabl to be the same as the configuration?